iapetus/data/dppc/myhydrogens.xml

```diff
@@ -0,0 +1,527 @@
+<Residues>
```

Comment:
Could you add a `README.md` file that describes how this file was constructed and what it contains?